app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -108,30 +108,7 @@ edit_link(
       <% end %>
       <%= cell "decidim/proposals/proposal_tags", @proposal %>
     </div>
-    <% if @proposal.answered? && translated_attribute(@proposal.answer).present? %>
-      <% if @proposal.accepted? %>
-        <div class="section">
-          <div class="callout success">
-            <h5><%= t(".proposal_accepted_reason") %></h5>
-            <p><%= decidim_sanitize translated_attribute @proposal.answer %></p>
-          </div>
-        </div>
-      <% elsif @proposal.rejected? %>
-        <div class="section">
-          <div class="callout warning">
-            <h5><%= t(".proposal_rejected_reason") %></h5>
-            <p><%= decidim_sanitize translated_attribute @proposal.answer %></p>
-          </div>
-        </div>
-      <% else %>
-        <div class="section">
-          <div class="callout secondary">
-            <h5><%= t(".proposal_in_evaluation_reason") %></h5>
-            <p><%= decidim_sanitize translated_attribute @proposal.answer %></p>
-          </div>
-        </div>
-      <% end %>
-    <% end %>
+    <%= cell("decidim/announcement", proposal_reason_callout_args) if @proposal.answered? %>
```

Comment:
This other one from here:
![image](https://user-images.githubusercontent.com/9702463/88786754-3357a100-d193-11ea-987f-107be983cd5b.png)
Does this override have been deleted ? Comparing with decidim/decidim they are equals.

Reply:
It wasn't deleted, the route is wrong, it's actually `app/views/decidim/proposals/proposals/show.html.erb` instead.

Reply:
Sorry, I meant that cell("decidim/announcement".... inside show.html.erb instead the code block deleted.
As I can see, due to the override, in the old file it was that big block:
![image](https://user-images.githubusercontent.com/9702463/88802993-1e3b3c00-d1ac-11ea-8aa8-bfdc9d5b217f.png)

and now has been replaced by that cell's block:
![image](https://user-images.githubusercontent.com/9702463/88802974-15e30100-d1ac-11ea-868f-3b73a52d51ea.png)

Comparing with the original's decidim/decidim template, I could see that's how the template actually is.
![image](https://user-images.githubusercontent.com/9702463/88803017-298e6780-d1ac-11ea-96d6-9be9dc54ac17.png)

Reply:
The only thing that the override changes is `strip_tags: false` instead of `true` https://github.com/CodiTramuntana/decidim-som-energia-app/blob/5a18b839b0b317558c52827a6a12a66509c3ae75/app/views/decidim/proposals/proposals/show.html.erb#L104
The cell comes from the source code of Decidim version 0.21-stable.